### ISSUE TITLE
anyone implement the baseline model?

### ISSUE THREAD

**Issue by w5688414** [minstone/HARRISON-Dataset#4]
I use keras to implement baseline model, but I got very low results, I donn't know why, I am wandering whether somebody can get the baseline source code or get the results illustrated in the paper

**Comment by tavi903** [comment]
Same here. Precision with K=1 is around 10%, accuracy with K=5 is around 28%. Both are far below the results in the original paper by the authors. I am using Pytorch, If someone manages to get results similar to the paper, I'd appreciate if they share them.
